nengo/builder/probe.py

```diff
@@ -33,10 +33,8 @@ def synapse_probe(model, key, probe):
         raise ValueError("Attribute '%s' is not probable on %s."
                          % (key, probe.obj))
 
-    if isinstance(probe.slice, slice):
+    if probe.slice is not None:
         sig = sig[probe.slice]
-    else:
```

Comment:
There was no way this `else` was ever being reached, and as far as I can tell indexing slices works fine, so I think this was just left in by accident?